apiserver/facades/client/machinemanager/machinemanager.go

```diff
@@ -465,9 +458,6 @@ func (mm *MachineManagerAPI) completeUpgradeSeries(arg params.UpdateSeriesArg) e
 	return machine.CompleteUpgradeSeries()
 }
 
-// [TODO](externalreality) We still need this, eventually the lock is going to cleaned up
-// RemoveUpgradeSeriesLock removes a series upgrade prepare lock for a
-// given machine.
 func (mm *MachineManagerAPI) removeUpgradeSeriesLock(arg params.UpdateSeriesArg) error {
```

Comment:
Do we need this at all? Doesn't look like it is called by anything.

Reply:
We need this because the CLI cleans up upon failure using this method. However, the way the CLI cleans seems as if it will never hit a point where it can fail after creating a lock. That may be changing really soon though.